### PR TITLE
[FIX] En el middleware de validationHandler.js

### PR DESCRIPTION
**Pull request by ealexisaraujo**
# En el middleware de validationHandler.js

## Description

En la nueva version de Joi ya no existe la funcion de joi.validate, ahora se hace de otra forma, estamos usando la version de Joi 16.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Hice los cambios en mi codigo original y funciona correctamente el cambio de linea

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: movies-api/utils/middleware/validationHandler.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ const boom = require('@hapi/boom');
 const joi = require('@hapi/joi');
 
 function validate(data, schema) {
-  const { error } = joi.validate(data, schema);
+  const { error } = joi.object(schema).validate(data);
   return error;
 }
 
-function validationHandler(schema, check = 'body') {
+functio  validationHandler(schema, check = 'body') {
```
Comment (glrodasz) [inline movies-api/utils/middleware/validationHandler.js:9]: Hay un typo aquí.

File: movies-api/utils/middleware/validationHandler.js [hunk 2]
```diff
@@ -2,11 +2,11 @@ const boom = require('@hapi/boom');
 const joi = require('@hapi/joi');
 
 function validate(data, schema) {
-  const { error } = joi.validate(data, schema);
+  const { error } = joi.object(schema).validate(data);
```
Comment (glrodasz) [inline movies-api/utils/middleware/validationHandler.js:5]: Si esto funciona con la nueva version de Joi, entonces el PR también debería actualizar el package.json y yarn.lock

File: movies-api/utils/middleware/validationHandler.js (hunk 1 above)
Comment (ealexisaraujo) [reply movies-api/utils/middleware/validationHandler.js]: Ya lo resolvi el typo @glrodasz

File: movies-api/utils/middleware/validationHandler.js (hunk 2 above)
Comment (ealexisaraujo) [reply movies-api/utils/middleware/validationHandler.js:5]: Listo actualice el package.json and package-lock.json con la nueva version de @hapi/joi 🙊

**Pull request merged**